AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using AllReady.ViewModels.Event;
+using MediatR;
+
+namespace AllReady.Features.Event
+{
+    public class EventByMonthQueryHandler : IRequestHandler<EventByMonthQuery, IEnumerable<EventViewModel>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public EventByMonthQueryHandler(IAllReadyDataAccess dataAccess)
+        {
```

Comment:
@danielepo, we're trying to phase out the usage of `IAllReadyDataAccess` (I believe it's been marked as obsolete in the latest master branch). For this handler, can you instead inject AllReadyContext and work directly against that? There are examples in the code of how to do this, as well as how to test it. Thanks!